package.json

```diff
@@ -29,7 +29,8 @@
 	"capabilities": {
 		"untrustedWorkspaces": {
 			"supported": "limited",
-			"restrictedConfigurations": ["biome.lspBin"]
+			"restrictedConfigurations": ["biome.lspBin"],
+			"description": "Binaries are Unavaible"
```

Comment:
```suggestion
			"description": "The use of external binaries is disallowed in untrusted workspaces."
```

Reply:
I'm not sure, but I think it will make the message redundant, hence my context-less description content.
Might test when I will have access to a computer.

Reply:
Can you point to the location where this message is displayed? I assumed it was metadata.

Reply:
@nhedger you can view a visual in the official docs here: https://code.visualstudio.com/api/extension-guides/virtual-workspaces#no-support-for-virtual-workspaces

Reply:
That one seems to be about `virtualWorkspaces`, not `untrustedWorkspaces` though?

Reply:
I think it's almost the same at the exception of the icon.

Here are the user docs and visuals about untrusted workspaces: https://code.visualstudio.com/docs/editor/workspace-trust#_extensions